test/missing-support.js

```diff
@@ -33,24 +33,50 @@ test('provides list of selected browsers', function (t) {
 test('for browser request ie >= 7, safari >= 6, opera >= 10.1', function (t) {
   var data = missingSupport([
     'ie >= 7',
-    'safari >= 6'
+    'safari >= 6',
+    'opera >= 10.1'
   ]).features
 
   var bgimgopts = data['background-img-opts']
-  hasKeys(t, bgimgopts, ['missing', 'title', 'missingData', 'caniuseData'])
+  hasKeys(t, bgimgopts, ['missing', 'partial', 'title', 'missingData', 'partialData', 'caniuseData'])
 
   var missing = bgimgopts.missingData
-  hasKeys(t, missing, ['ie', 'safari'])
+  var partial = bgimgopts.partialData
+  hasKeys(t, missing, ['ie'])
   hasKeys(t, missing.ie, ['7', '8'])
-  hasKeys(t, missing.safari, ['6', '6.1'])
+  hasKeys(t, partial, ['safari', 'opera'])
+  hasKeys(t, partial.safari, ['6', '6.1'])
+  hasKeys(t, partial.opera, ['10.0-10.1'])
+  t.end()
+})
+
+test('only yields features partially supported by selected browser', function (t) {
```

Comment:
I think this description (and the one for the old test below) is now a bit confusing.  Could we maybe prepend `partialData` to the description, so that it reads, "partialData only yields features partially supported by selected browser" (and likewise with `missingData` for the one below)?